packages/uxpin-merge-cli/src/steps/serialization/component/comments/__tests__/parseUsePortal.test.ts

```diff
@@ -0,0 +1,23 @@
+import { parseAsCondition } from '../parseUsePortal';
+
+describe('Parse string and check valid condition (to be used in a function run against props)', () => {
```

Comment:
this parseAsCondition(`props.isValid && props.mode === "modal" true false`)
returns true but tbh I'm not sure that we have to support this case :)

Reply:
```props.isValid && props.mode === "modal"; alert(1)```
and this pass tests as well :hacker:

Reply:
Yes, like I said it it's pretty basic, but I can improve by rejecting multiple statements (there should be only one `body`) as the logic written by the user should very simple: a condition and nothing more, I will do that 👍